docker/wasm-build.Dockerfile

```diff
@@ -17,9 +17,17 @@ ENV RUST_BACKTRACE="short"
 ENV env=$env
 COPY . .
 RUN echo env
```

Comment:
Nit: Do we need the `ENV env=$env` and the `RUN echo env` lines? If not, can we remove them?